marklogic-data-hub/src/test/java/com/marklogic/hub/dhs/DeployUserAmpsTest.java

```diff
@@ -0,0 +1,98 @@
+package com.marklogic.hub.dhs;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ArrayNode;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.marklogic.appdeployer.command.security.DeployAmpsCommand;
+import com.marklogic.hub.AbstractHubCoreTest;
+import com.marklogic.hub.HubConfig;
+import com.marklogic.hub.impl.Versions;
+import com.marklogic.mgmt.resource.security.AmpManager;
+import com.marklogic.rest.util.ResourcesFragment;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import java.io.IOException;
+import java.nio.file.Path;
+
+
+public class DeployUserAmpsTest extends AbstractHubCoreTest {
+    ObjectMapper mapper = new ObjectMapper();
+
+    @BeforeEach
+    void checkIfTestsCanBeRun(){
+        Versions.MarkLogicVersion mlVersion = versions.getMLVersion();
+        Assertions.assertTrue(mlVersion.isNightly() || (mlVersion.getMajor() == 10 && mlVersion.getMinor() >= 440));
+    }
+
+    @Test
+    void createAmpWithInheritableRole() {
+        //POST request
+        writeAmpFileToProjectAndDeploy(HubConfig.DEFAULT_MODULES_DB_NAME, "data-hub-developer");
+        ResourcesFragment amps = new AmpManager(runAsDataHubSecurityAdmin().getManageClient()).getAsXml();
+        Assertions.assertTrue(amps.resourceExists("testAmp"));
+
+        //PUT request(doesn't work)
+        writeAmpFileToProjectAndDeploy(HubConfig.DEFAULT_MODULES_DB_NAME,"data-hub-developer", "data-hub-operator");
+
+        //DELETE request(doesn't work)
+        new DeployAmpsCommand().undo(newCommandContext(runAsDataHubSecurityAdmin()));
```

Comment:
Try doing a delete request with a ManageClient - I recall that the URL is a bit funky to build

Reply:
Deleting with ManageClient seems to work